lib/adapters/REST/endpoints/space-team.ts

```diff
@@ -0,0 +1,18 @@
+import type { AxiosInstance } from 'contentful-sdk-core'
+import { CollectionProp, GetSpaceTeamParams, PaginationQueryParams } from '../../../common-types'
+import { SpaceTeamProps } from '../../../entities/space-team'
+import { RestEndpoint } from '../types'
+import * as raw from './raw'
+
+export const get: RestEndpoint<'SpaceTeam', 'get'> = (
+  http: AxiosInstance,
+  params: GetSpaceTeamParams & { teamId: string }
+) => raw.get<SpaceTeamProps>(http, `/spaces/${params.spaceId}/teams/${params.teamId}`)
```

Comment:
@matthew-contentful do we actually have an endpoint for this? 🤔 I thought we only introduced `/spaces/${params.spaceId}/teams`

Reply:
We do, rails made it easier to leave that in than to remove it. It returns a single team object, not an array or any other payload metadata.

Reply:
And oops, I need to add it to the customer facing docs :)

Reply:
Ok this is good news, then we can keep it

Reply:
As discussed with Matthew, we will hide this endpoint completely for now. Gonna remove it in cma and docs